include/experimental/__p0009_bits/macros.hpp

```diff
@@ -118,7 +118,9 @@ MDSPAN_FUNCTION inline void default_precondition_violation_handler(const char* c
 #elif defined(MDSPAN_IMPL_HAS_SYCL)
 MDSPAN_FUNCTION inline void default_precondition_violation_handler(const char* cond, const char* file, unsigned line)
```

Comment:
All arguments are unused if the compiler isn't `IntelLLVM` which might result in compiler warnings.

Reply:
Good point.
Do you prefer me to `(void) foo` them or to use some attribute to disable warning for this function / argument?

Reply:
Looks like this file uses `(void) bla` instead of `[[maybe_unused]]`.

Reply:
Sorry, I'm stupid. I should have just looked too.

Reply:
Fixed with b86c4f1c6955e2ff6bdd3abec048dc1c846527a1